independent-projects/arc/runtime/src/main/java/io/quarkus/arc/impl/SuperclassInvocationContext.java

```diff
@@ -0,0 +1,58 @@
+package io.quarkus.arc.impl;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.List;
+import java.util.function.BiFunction;
+
+import jakarta.interceptor.InvocationContext;
+
+/**
+ * A special {@link javax.interceptor.InvocationContext} that is used if multiple interceptor methods are declared in a
+ * hierarchy of an interceptor class.
+ * <p>
+ * The interceptor methods defined by the superclasses are invoked before the interceptor method defined by the interceptor
+ * class, most general superclass first.
+ */
+class SuperclassInvocationContext extends InnerInvocationContext {
```

Comment:
This doesn't seem to be used anywhere?

Reply:
Sorry, please ignore -- I missed `InvocationContexts.performSuperclassInterception`.